apps/frontend/src/locales/en/translations.json

```
@@ -552,5 +553,42 @@
         "changelog": "See changelog <0>here</0>",
         "description": "Click below to update and accept the new version.",
         "updateButton": "Update"
+    },
+    "emailNotifications": {
+        "errorPage": {
+            "meta": {
+                "title": "Error - Sovryn"
+            },
+            "title": "Something went wrong!",
+            "subtitle": "Please try the action again. If it continues to not work, please contact a support agent at <0>link</0>"
+        },
+        "verifyPage": {
+            "meta": {
+                "title": "Verify email - Sovryn"
```

```suggestion
                "title": "Verified - Sovryn"
```

Incorrect as per spec

---

it's already fixed, just need to push my updates